packages/block-library/src/navigation/edit.js

```diff
@@ -148,17 +158,32 @@ function Navigation( {
 		);
 	}
 
+	const blockClassNames = classnames( 'wp-block-navigation', {
+		[ `items-justification-${ attributes.itemsJustification }` ]: attributes.itemsJustification,
+	} );
+
 	// UI State: rendered Block UI
 	return (
 		<Fragment>
 			<BlockControls>
+				<Toolbar
+					icon={ attributes.itemsJustification ? `editor-align${ attributes.itemsJustification }` : 'editor-alignleft' }
+					label={ __( 'Change items justification' ) }
+					isCollapsed
+					controls={ [
+						{ icon: 'editor-alignleft', title: __( 'Justify items left' ), isActive: 'left' === attributes.itemsJustification, onClick: handleItemsAlignment( 'left' ) },
```

Comment:
We could consider using a different set of icons, closer to what we use for columns vertical alignment. cc @jasmussen 

![image](https://user-images.githubusercontent.com/548849/70171238-98bfc480-16ce-11ea-9540-4a53132778a5.png)

Reply:
Yes that's an excellent point, as you're not aligning _text_.

Reply:
I think we should but could someone make some for me please? 😁

Reply:
Can you reuse the ones from the columns block?

![Screenshot 2019-12-04 at 20 29 09](https://user-images.githubusercontent.com/1204802/70174347-ff47e100-16d4-11ea-89a7-6e20670ccd4d.png)

or wait, maybe you mean horizontal versions, right?

I'll get them here for you tomorrow if that's okay.

Reply:
Here you go. Left:

```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M11 16v-3h10v-2H11V8l-4 4 4 4zM5 4H3v16h2V4z"/></svg>
````

Center:

```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M5 8v3H1v2h4v3l4-4-4-4zm14 8v-3h4v-2h-4V8l-4 4 4 4zM13 4h-2v16h2V4z"/></svg>
```

Right:
```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 8v3H3v2h10v3l4-4-4-4zm8-4h-2v16h2V4z"/></svg>
```

Reply:
Super helpful! Thank you

<img width="499" alt="Screen Shot 2019-12-05 at 09 09 34" src="https://user-images.githubusercontent.com/444434/70220777-04da1100-173f-11ea-8d2e-eed9f5e5e456.png">